Adding `meta_desc` to blogs (first batch) (#1499)
pulumi/docs: content/blog/building-a-future-of-cloud-engineering/index.md

title: "Building a future of cloud engineering"
authors: ["joe-duffy"]
tags: ["Pulumi-News"]

> meta_desc: "Using your favorite general purpose programming language and express the entirety of your cloud software. Program the cloud with Pulumi."

Reviewer: ```suggestion
meta_desc: "Using your favorite general purpose programming language to define your cloud infrastructure and applications. Program the cloud with Pulumi."
```
